### PR TITLE
Unipilot token balance [unipilot-token-balance]

### PR DESCRIPTION
**Pull request by RAFAQAT11**
Fixes # .

Changes proposed in this pull request:
- with this pull request unipilot user will be able to vote regardless of their tokens are staked.

### CODE REVIEW

File: src/strategies/index.ts [hunk 1]
```diff
@@ -377,7 +378,8 @@ const strategies = {
   'protofi-erc721-tier-weighted': protofiErc721TierWeighted,
   'erc721-with-tokenid': erc721WithTokenId,
   'erc721-with-tokenid-range-weights': erc721WithTokenIdRangeWeights,
-  'erc721-with-tokenid-range-weights-simple': erc721WithTokenIdRangeWeightsSimple,
+  'erc721-with-tokenid-range-weights-simple':
+    erc721WithTokenIdRangeWeightsSimple,
```
Comment (ChaituVR) [inline src/strategies/index.ts:382]:
```suggestion
'erc721-with-tokenid-range-weights-simple': erc721WithTokenIdRangeWeightsSimple,
```

File: src/strategies/index.ts [hunk 2]
```diff
@@ -573,7 +575,8 @@ const strategies = {
   'snet-stakers': snetStakers,
   'snet-liquidity-providers': snetLiquidityProviders,
   'minmax-mcn-farm': minMaxMcnFarm,
-  'unstackedtoadz-and-stackedtoadz-stakers': unstackedToadzAndStackedToadzStakers,
+  'unstackedtoadz-and-stackedtoadz-stakers':
+    unstackedToadzAndStackedToadzStakers,
```
Comment (ChaituVR) [inline src/strategies/index.ts:579]:
```suggestion
  'unstackedtoadz-and-stackedtoadz-stakers': unstackedToadzAndStackedToadzStakers,
```

File: src/strategies/unipilot-token-balance/index.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        name: 'lastUpdateRewardToken',
+        type: 'uint256'
+      },
+      { internalType: 'uint256', name: 'amount', type: 'uint256' },
+      { internalType: 'uint256', name: 'rewardDebt', type: 'uint256' }
+    ],
+    stateMutability: 'view',
+    type: 'function'
+  }
+];
```
Comment (ChaituVR) [inline src/strategies/unipilot-token-balance/index.ts:28]:
```suggestion
const abi_pilot_staking = [
  'function userInfo(address) view returns (uint256 lastUpdateRewardToken, uint256 amount, uint256 rewardDebt)'
];
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @RAFAQAT11 I feel like you can use two `contract-call` strategies here https://snapshot.org/#/strategy/contract-call

**Comment by RAFAQAT11** [comment]
Hi @ChaituVR , suggested things has been implemented. Thanks.

**Pull request closed without merging**